src/app/screens/Login/index.js

```diff
@@ -33,11 +33,20 @@ class LoginContainer extends Component {
         !signInErrorMessage && navigation.replace(Home);
     };
 
+    async componentDidMount() {
+        const { authInit, navigation } = this.props;
+        const user = await AuthService.authSetup();
```

Comment:
you should only call actions in here that calls services. Replace this with a redux action and move the service inside there

Reply:
This will be include in the next PR.